System.Activities/System/Activities/Debugger/StateManager.cs

```diff
@@ -681,7 +681,7 @@ MethodBuilder CreateIsland(TypeBuilder typeBuilder, State state, bool withPrimin
             [SecurityCritical]
             void InitDynamicModule(string asmName)
             {
-                // See http://blogs.msdn.com/Microsoft/archive/2005/02/03/366429.aspx for a simple example
```

Comment:
I'm guessing that this is supposed to be: http://blogs.msdn.com/b/jmstall/archive/2005/02/03/366429.aspx. This is a good one to keep as-is.